extensions/cornerstone/src/Viewport/Overlays/CustomizableViewportOverlay.css

```diff
@@ -0,0 +1,21 @@
+/* custom overlay panels: top-left, top-right, bottom-left and bottom-right */
+.viewport-overlay {
+  max-width: 40%;
+}
+.viewport-overlay span {
+  max-width: 100%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
If any text to be displayed on the overlay is too long to hold on a single line, it will be truncated with ellipsis in the end.